docs/restrict-dynamic-log-locations.md

```diff
@@ -23,13 +23,101 @@ The benefits of restricting dynamic log locations in MySQL outweigh the disadvan
 
 ## secure_log_path
 
-The variable is read-only and must be set up in a configuration file or the command line.
+| Variable Name | Description       |
+|---------------|-------------------|
+| Command-line  | --secure-log-path |
+| Dynamic       | No                |
+| Scope         | Global            |
+| Data type     | String            |
+| Default       | empty string      |
 
-The expected value is the directory name provided as a string. The default value is an empty string. 
+The `secure_log_path` variable controls where specific log files are stored. The expected value for this variable is a directory name as a string. By default, it is an empty string.
+
+This variable affects the following options:
+
+| Option                        | Description                                           |
+|-------------------------------|-------------------------------------------------------|
+| `slow_query_log=ON`            | Enables the storage of `slow_query_log_file`.         |
+| `slow_query_log_file`          | Sets the name and location of the slow query log file. |
+|                                |                                                        |
+| `general_log=ON`               | Enables the storage of `general_log_file`.            |
+| `general_log_file`             | Sets the name and location of the general log file.   |
+|                                |                                                        |
+| `buffered_error_log=ON`        | Enables the storage of `buffered_error_log_filename`. |
```

Comment:
Enables the storage of `buffered_error_log_filename`.
sounds odd 
maybe
Enables the storage of the buffered error log file.